app/views/notes/_show.html.erb

```diff
@@ -1,6 +1,6 @@
 <%# locals: { note } %>
 <% if !note.nil? %>
-    <% user = User.find(note.user_id) %>
+    <% user = note.user %>
```

Comment:
good catch, no need to go back to the DB to find something that is already loaded